views/artist-search-results.hbs

```diff
@@ -0,0 +1,10 @@
+<h4>Artist Search Result</h4>
+<div class="artist-data">
+  {{#each this}}
+    <div class="artist-search">
+      <img src="{{images}}">
+      <p>Name: {{name}}</p>
+      <button class="submit-btn" href="/albums/{{uri}}"> View Album</button>
```

Comment:
Here, we want to use the artist id, in order to dispplay all albums.

href="/albums/{{this.id}}"